### ISSUE TITLE
Contact form gets submitted even if all the fields are empty

### ISSUE THREAD

**Issue by AsavariA** [adrianhajdin/portfolio_website#39]
Contact form gets submitted even if all the fields are empty, that is not a great user experience. It would be better if there were snackbars raising empty fields issue.

**Comment by amangupta679** [comment]
> Contact form gets submitted even if all the fields are empty, that is not a great user experience. It would be better if there were snackbars raising empty fields issue.

Yes, you're absolutely right. Having a contact form that gets submitted even if all the fields are empty is not a good user experience. Users might assume that their message was sent successfully, only to later realize that the form was never submitted due to missing fields.

Adding snackbars that raise empty fields issue is a great solution to this problem. When the user clicks the submit button, the form should first check if all the required fields are filled. If any field is empty, a snackbar should appear indicating that the user needs to fill all the required fields before submitting the form.

This provides a clear indication to the user that they need to provide all the required information, making it less likely that they will accidentally submit the form without filling in all the necessary fields. Additionally, this approach can help to reduce the number of form submissions that fail due to missing information, improving the overall user experience.

**Comment by amangupta679** [comment]
..

**Comment by lohithasiripurapu** [comment]
Hi @AsavariA @amangupta679 I can work on this issue, Can you assign it to me?

**Comment by ntehseen** [comment]
hi, I have been trying to render the <NavDropDown/> component but so far no luck. can anyone help me out? Thanks in advance 😊
